data/map.txt

```diff
@@ -26752,6 +26779,26 @@ planet Starcross
 		threshold 2500
 		fleet "Small Militia" 22
 
+planet "Starting Rubin"
+	attributes kimek "coalition station" shipping wealthy
+	landscape land/space3
+	description `Starting Rubin was far from being the first station constructed around the Kimek homeworld, but it is the oldest one to still be inhabited. It was built as an orbital equivalent to the hunger towers found on the planet it orbits, as a safety measure against the disasters that may affect Ki Patek Ka.`
+	description `Since the residents need few of the food produced, it soonly became the station's primary export, transforming it in one of the largest trading hubs of the region. It has also the largest population of every Coalition station, capable of supporting tens of thousands of Kimek.`
```

Comment:
`Since the residents need few of the food...` -> `Since the residents need little of the food...` (few -> little)
`it soonly became the station's primary export` -> `it quickly became the station's primary export` (soonly -> quickly)
`transforming it in one of the largest` -> `transforming it into one of the largest` ( in -> into)
`largest population of every coalition station` -> `largest population of any coalition station` (every - > any)

Reply:
Fixed, thank you